packages/utils/aria.ts

```diff
@@ -120,9 +120,9 @@ const Utils = {
    */
   focusFirstDescendant: function(element: HTMLElement): boolean {
     for (let i = 0; i < element.childNodes.length; i++) {
-      const child = element.childNodes[i]
+      const child = element.childNodes[i] as HTMLElement
       if (
-        attemptFocus(child as HTMLElement) ||
```

Comment:
what reason did you have on changing this?

Reply:
````
if (attemptFocus(child as HTMLElement) ||  this.focusLastDescendant(child) ) {
   return true
}
````

`focusLastDescendant` got warning: `Argument type ChildNode is not assignable to parameter type HTMLElement `

so change to 

````
const child = element.childNodes[i] as HTMLElement
````